HoloPipelines/pipelineServer.py

```diff
@@ -0,0 +1,202 @@
+from flask import Flask, request
```

Comment:
Rather than having them on the top level, could you move this file and `pipelineController` to a new directory/module `HoloPipelines/core`